src/app/services/login.service.ts

```diff
@@ -7,6 +7,7 @@ import { AuthResponse } from '../models/authResponse';
 
 @Injectable()
 export class LoginService {
+  server_id:string =''
```

Comment:
This doesn't seem to be used.

Reply:
It is used in HttpRequestsInterceptor
` const server_id = this.loginService.server_id;`

Reply:
It is used in HttpRequestsInterceptor file
` const server_id = this.loginService.server_id;`